packages/os/os.spec

```diff
@@ -424,6 +428,11 @@ Conflicts: %{_cross_os}bash
 %description -n %{_cross_os}brush
 %{summary}.
 
+%package -n %{_cross_os}configure-snapshotter
+Summary: Configure snapshotter service
+%description -n %{_cross_os}configure-snapshotter
+Service to configure snapshotter and clean up containerd and soci-snapshotter-grpc directories when snapshotter changes.
```

Comment:
It is convention in bottlerocket to use the `summary` as the description of the package.